yt_dlp/extractor/rokfin.py

```diff
@@ -210,6 +372,8 @@ class RokfinChannelIE(RokfinPlaylistBaseIE):
             'description': 'md5:bb622b1bca100209b91cd685f7847f06',
         },
     }]
+    IE_NAME = 'rokfin:channel'
+    _NETRC_MACHINE = False
```

Comment:
why?

Reply:
Without `NETRC_MACHINE = False`, line 542 in common.py prints

> WARNING: [rokfin:channel] Login with password is not supported for this website. Use --cookies-from-browser or --cookies for the authentication. See  https://github.com/ytdl-org/youtube-dl#how-do-i-pass-cookies-to-youtube-dl  for how to manually pass cookies

Reply:
Well, the warning is correct, no? Why do you think it can be suppressed?

Reply:
Does Rokfin not support logins with passwords? Did I miss something?

Reply:
`rokfin:channel` does not. If you can use the same login code for it, move it (along with `NETRC_MACHINE` to a base class)

Reply:
Ok, now I can see what the warning says. But it's still really misleading to say "Login with password is not supported for this **website**".